### DISCUSSION TITLE
I didn't get my benefits still it's being 3 days

### DISCUSSION THREAD

**Discussion by VenxPlays** [community/community#134295] [GitHub Education]
### Hi everyone,

I didn't get my benefits of student developer pack it's has been 3 days

**Comment by davevad93** [comment]
Hi @VenxPlays ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @VenxPlays was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
